lib/actions/ui.js

```diff
@@ -78,7 +78,17 @@ export function matchContentToUrl (location) {
         // Update map location/zoom and optionally override router ID.
         dispatch(setMapCenter({ lat, lon }))
         dispatch(setMapZoom({ zoom }))
-        if (routerId) dispatch(setRouterId(routerId))
+        // If router ID is provided, override the default routerId.
+        if (routerId) {
+          // This is a somewhat hidden element of the trip planner, so show a
+          // confirmation message to the user to ensure they are aware of what
+          // is happening.
+          const useAltRouterId = window.confirm(
+            `The trip planner's router is set to ${routerId}.\n\nClick OK to confirm or Cancel to use the default router.\n\nNote: new sessions opened with URLs copied from this tab's address bar will not reference the correct router (must contain the #/start/lat/lon/zoom/router prefix).`
```

Comment:
Suggestion for future work: we might want to show this in the nav bar so it is less intrusive.

Reply:
Agree that this is a highly technical change in a variable that most users probably won't understand. Since this variable is already in the URL, do we even need to confirm this anyways?